addons/html_editor/controllers/main.py

```diff
@@ -570,15 +570,21 @@ def link_preview_metadata_internal(self, preview_url):
 
             record_id = int(words.pop())
             action_name = words.pop()
-            action = Actions.sudo().search([('path', '=', action_name)])
-            if not action:
-                return {'error_msg': _("Action %s not found, link preview is not available, please check your url is correct", action_name)}
-            action_type = action.type
-            if action_type != 'ir.actions.act_window':
-                return {'other_error_msg': _("Action %s is not a window action, link preview is not available", action_name)}
-            action = request.env[action_type].browse(action.id)
-
-            model = request.env[action.res_model].with_context(context)
+            if (action_name.startswith('m-') or '.' in action_name) and action_name in request.env and not request.env[action_name]._abstract:
+                """if path format is `odoo/<model>/<record_id>` so we use `action_name` as model name"""
```

Comment:
Is this supposed to be a comment? Because right now, it's a string that just floats there in the middle of the code... ^^' I looked in Odoo's codebase (I haven't worked in Python for years) and it seems this kind of triple quoted strings are only used as comments for the docstring of functions (as per [PEP 257](https://peps.python.org/pep-0257/)), that is right after the function definition, but never in the middle of the code itself. Classical Python comments use `#`.

Reply:
Of course, i'm sorry yea it should be a comment not a string.